sources/procfs.go

```diff
@@ -531,9 +530,7 @@ func parseStatsFile(helpText string, promName string, path string) (metricList [
 		return nil, err
 	}
 	if statsList != nil {
-		for _, item := range statsList {
-			metricList = append(metricList, item)
-		}
+		metricList = append(metricList, statsList...)
```

Comment:
See my other comment - we do the opposite here. Might be good to stay consistent.